### PR TITLE
#74 issue page view and styles

### PR DESCRIPTION
**Pull request by mis6ko13**
Issue - https://github.com/Kotany-IF070-WebUI-RoR/NewVasjuky/issues/74
- [ ] @Kmalynss 
- [ ] @igorkasyanchuk 
- [ ] @krisberry
- [ ] @f1nwe 

Added views and styles for issues/:id page;
Added styles to comments;
Minor fixes and refactoring.

### CODE REVIEW

File: app/controllers/issues_controller.rb [hunk 1]
```diff
@@ -37,4 +41,18 @@ def issues_params
                                   :category_id, :description, :attachment,
                                   :location, :title)
   end
+
+  def find_relevant(category_id, issue_id)
+    Issue.where(category_id: category_id)
+         .where.not(id: issue_id)
+  end
+
+  def issue_status(status)
```
Comment (krisberry) [inline app/controllers/issues_controller.rb]: unsuitable method name, this method returns label but not issue status

File: app/controllers/issues_controller.rb [hunk 2]
```diff
@@ -12,8 +12,12 @@ def new
 
   def show
     @issue = Issue.find(params[:id])
+    category_id = @issue.category_id
```
Comment (igorkasyanchuk) [inline app/controllers/issues_controller.rb]: @relevant_issues = @issue.category.issues.where.not(id: issue_id).order('random()').limit(4)

Comment (igorkasyanchuk) [inline app/controllers/issues_controller.rb]: + approved (or what name you have) ... + to show pending

File: app/controllers/issues_controller.rb [hunk 3]
```diff
@@ -37,4 +41,18 @@ def issues_params
...
                                   :location, :title)
   end
+
+  def find_relevant(category_id, issue_id)
+    Issue.where(category_id: category_id)
+         .where.not(id: issue_id)
+  end
+
+  def issue_status(status)
+    case status
```
Comment (igorkasyanchuk) [inline app/controllers/issues_controller.rb]: this is must be a helper class.
AND IT WAS IMPLEMENTED IN SOME PR (ON ISSUES LIST)

WORK TOGHETHER AS A TEAM

File: app/controllers/issues_controller.rb (hunk 1 above)
Comment (igorkasyanchuk) [inline app/controllers/issues_controller.rb]: remove this method

**Pull request merged**